orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/support/TargetServerGroup.groovy

```diff
@@ -73,6 +76,14 @@ class TargetServerGroup {
     return (serverGroup.instances ?: []) as List<Map>
   }
 
+  /**
+   * Used in TrafficGuard, which is Java, which doesn't play nice with @Delegate
+   */
+  Moniker getMoniker() {
+    ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Should inject an `ObjectMapper` into the constructor (it'll be autowired) vs explicitly creating one here.